src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,39 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static int SIZE_OF_ARRAY = 10;
+    private Pair[] pairs = new Pair[SIZE_OF_ARRAY];
```

Comment:
Make a variable final, parameterized, and declare pair into the contract 
```suggestion 
      private final Pair<K,V>[] pairs;
      
      public StorageImpl() {
        pairs = new Pair<>[SIZE_OF_ARRAY];
      }
```